src/ZEO/asyncio/tests.py

```diff
@@ -1,12 +1,3 @@
-from .._compat import PY3
-
-if PY3:
-    def to_byte(i):
-        return bytes([i])
-else:
-    def to_byte(b):
-        return b
-
 from zope.testing import setupstack
 import mock
```

Comment:
While you're at it you should replace all use of `mock` with `unittest.mock` and remove the dependency.

Reply:
Right, @dataflake. I was going to do that next, after we agree on the approach, but since you are asking for it now, it goes in hereby pull-request as a third patch.